packages/plugins/src/plugins/stakewise/lendingFetcher.ts

```diff
@@ -0,0 +1,96 @@
+import { getAddress } from 'viem';
+import { ethereumNativeAddress } from '@sonarwatch/portfolio-core';
+import { Fetcher, FetcherExecutor } from '../../Fetcher';
+
+import { Cache } from '../../Cache';
+import {
+  OS_TOKEN_ADDRESS,
+  PLATFORM_ID,
+  STAKEWISE_VAULTS_CACHE_PREFIX,
+  StakewiseSupportedNetworkIds,
+} from './constants';
+import { getEvmClient } from '../../utils/clients';
+import { stakewiseVaultABI } from './abis';
+import { ethFactorBigInt } from '../../utils/evm/constants';
+import { zeroBigInt } from '../../utils/misc/constants';
+import { ElementRegistry } from '../../utils/elementbuilder/ElementRegistry';
+import { StakewiseVault } from './types';
+
+export function getLendingFetcher(
+  networkId: StakewiseSupportedNetworkIds
+): Fetcher {
+  const executor: FetcherExecutor = async (owner: string, cache: Cache) => {
+    const ownerAddress = getAddress(owner);
+    const client = getEvmClient(networkId);
+
+    const enrichedVaults = await cache.getItem<StakewiseVault[]>(
+      STAKEWISE_VAULTS_CACHE_PREFIX,
+      {
+        networkId,
+        prefix: STAKEWISE_VAULTS_CACHE_PREFIX,
+      }
+    );
+    if (!enrichedVaults?.length) {
+      return [];
+    }
+
+    const userVaultPositionResponses = await client.multicall({
+      contracts: enrichedVaults.flatMap((vault) => [
```

Comment:
I was told on some of my prs to do 
```suggestion
      contracts: enrichedVaults.flatMap<ContractFunctionConfig<typeof stakewiseVaultABI>>((vault) => [
```